e2e/src/test/java/com/arcadedb/e2e/JdbcQueriesTest.java

```diff
@@ -131,6 +132,8 @@ void createVertexCypherQuery() throws Exception {
   }
 
   @Test
+  @Disabled
```

Comment:
this test runs without errors on my side and on CI too.

Reply:
I am using OpenJVM 17.

Should I just undo this change?